html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>HTML Forms</title>
+    </head>
+    <body>
+        <h1>Coding Form!</h1>
+        <form action="" method="get">
+            <div>
+                <label for="email"> Email: <br></label>
+                    <input type="email" name="email"/> <br>
+            </div>
+            <div>
+                <label for="select">
```

Comment:
This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.